lesson28/js/task_2.js

```diff
@@ -0,0 +1,48 @@
+// 2) Реализовать класс, описывающий простой маркер. В классе должны быть следующие компоненты:
+
+// поле, которое хранит цвет маркера;
+// поле, которое хранит количество чернил в маркере (в процентах);
+// метод для печати (метод принимает строку и выводит текст соответствующим
+//      цветом; текст выводится до тех пор, пока в маркере есть чернила;
+//      один не пробельный символ – это 0,5% чернил в маркере).
+// Реализовать класс, описывающий заправляющийся маркер, унаследовав его
+// от простого маркера и добавив метод для заправки маркера.
+
+// Продемонстрировать работу написанных методов.
+
+class Marker {
+  constructor(color) {
+    (this.color = color), (this.amount = 100);
+  }
+  printToScreen(string) {
+    let html = "";
+    for (let char of string) {
+      if (this.amount === 0) {
+        break;
+      }
+      if (char !== " " && this.amount !== 0) {
```

Comment:
ти вже виключив цю умову вище
```suggestion
      if (char !== " ") {
```

Reply:
Ступив, згідний)